cmake/TPLs/FindTPLTrilinos.cmake

```diff
@@ -0,0 +1,13 @@
+FIND_PACKAGE(Trilinos REQUIRED
+    CONFIG
+    HINTS
+      ${TPL_Trilinos_DIR}/lib/cmake/Trilinos
+      ${TPL_Trilinos_DIR}
+    COMPONENTS
+      ${${PACKAGE_NAME}_Trilinos_REQUIRED_COMPONENTS}
+    OPTIONAL_COMPONENTS
+      ${${PACKAGE_NAME}_Trilinos_OPTIONAL_COMPONENTS}
+)
+GLOBAL_SET(TPL_Trilinos_INCLUDE_DIRS "${Trilinos_INCLUDE_DIRS};${Trilinos_LIBRARY_DIRS}")
+GLOBAL_SET(TPL_Trilinos_LIBRARIES    "${Trilinos_LIBRARIES};${Trilinos_TPL_LIBRARIES}")
+GLOBAL_SET(TPL_Trilinos_LIBRARY_DIRS "${Trilinos_LIBRARY_DIRS};${Trilinos_TPL_LIBRARY_DIRS}")
```

Comment:
Add comment that you stole that logic from that TriBITS macro for registering TPLs that are marked as `TRIBITS_PKG`.

Reply:
I didn't. I just added these lines after TriBITS complained about them missing.